hcp/io/tests/test_io.py

```diff
@@ -79,7 +79,6 @@ def test_read_raw_task():
             _basic_raw_checks(raw=raw)
 
 
-@expensive_test
```

Comment:
I looked at the `keys` stuff in `tests/config.py` and it looks like the empty-room data should actually be in there. Any idea how to get it to work?